src/genpy/message.py

```diff
@@ -130,8 +131,10 @@ def strify_message(val, indent='', time_offset=None, current_time=None, field_fi
         if type(val0) in (int, float) and fixed_numeric_width is not None:
             list_str = '[' + ''.join(strify_message(v, indent, time_offset, current_time, field_filter, fixed_numeric_width) + ', ' for v in val).rstrip(', ') + ']'
             return list_str
-        elif type(val0) in (int, float, str, bool):
-            # TODO: escape strings properly
+        elif isstring(val0):
+            # escape list of strings for use in yaml file using yaml dump
+            return yaml.dump(val).rstrip('\n')
```

Comment:
no, its not needed here since val is a list of strings which will never be serialized using the "\n...\n" format. (this format is used only for "simple" strings, with no characters that need escaping...)